docs/developers-guide/cadl-docs/README.md

```diff
@@ -2,158 +2,321 @@
 ### SDK Types to CADL
 
 #### Basic Types
-##### [Bool](https://package.elm-lang.org/packages/elm/core/latest/Basics#Bool) 
-Boolean, a `true` or `false` value in morphir, maps directly to the `boolean` type in CADL. 
+##### [Bool](https://package.elm-lang.org/packages/elm/core/latest/Basics#Bool)
+Bool, a `true` or `false` value in morphir, maps directly to the `boolean` type in CADL.
+
+Elm:
+```elm
+type alias isValid = 
+    Bool
 ```
-alias isValid = boolean
+Cadl:
+```cadl
+alias isValid = boolean;
 ```
-        
+
 ##### [Int](https://package.elm-lang.org/packages/elm/core/latest/Basics#Int)
-In CADL, this maps to the subtype `integer`. The `integer` type assignment is valid CADL, but <br/>
-<span style="color: red; font-style:italic;"> Things to note :: </span>
-1. When dealing with emitters such as OpenApiSpec(OAS) it defaults to an object. To obtain an actual int value, specify a subtype `int64`.
+The `Int` type in morphir is a set of natural numbers(positive and negative) without a fraction component, maps directly to the `integer` type in cadl.
+
+Elm:
+```elm
+type alias Foo = 
+    Int
 ```
+Cadl:
+```cadl
 alias Foo = integer;
+```
+<span style="color: red; font-style:italic;"> Note: </span>
 
-// when dealing with emitters
+The `integer` type assignment is valid CADL, but when dealing with emitters such as OpenApiSpec(OAS) it defaults to an object. To obtain an actual int value, specify the subtype `int64`.
+```cadl
 alias Foo = int64;
-                              |       }
 ```
 
 ##### [Float](https://package.elm-lang.org/packages/elm/core/latest/Basics#Float)
-The `float` type in morphir, maps directly to subtype `float` in CADL. <span style="color: red; font-style:italic;">Same issues with `integer` type is applicable </span>
- 
+The `Float` type; floating point number, in morphir maps directly to the `float` type in CADL.
+
+Elm:
+```elm
+type alias Pi = 
+    Float
+```
+Cadl:
+```cadl
+alias Pi = float;
 ```
-alias PI = float
+<span style="color: red; font-style:italic;">Note: </span>
 
-// when dealing with emitters
-alias Foo = float64;
-                             |       }
+The `float` type assignment is valid CADL, but when dealing with emitters such as OpenApiSpec(OAS) it defaults to an object. To obtain an actual float value, specify the subtype `float64`.
```

Comment:
What does this actually mean? Given that this is a specification for how Morphir maps to CADL, we shouldn't be talking about how things should be done. We need to say exactly what our tooling does. Maybe we should revisit our mapping here?